addon/query-cache.js

```diff
@@ -2,6 +2,7 @@ import { Promise as RSVPPromise } from 'rsvp';
 import { assert } from '@ember/debug';
 import { get } from '@ember/object';
 import { assign } from '@ember/polyfills';
+import { serializeQueryParams } from '@ember-data/adapter/-private';
```

Comment:
Should we port this util function instead of adding a new dependency?

Reply:
yes

1. we definitely don't want a dependency on `@ember-data/adapter`
2. this is also the reason the tests are failing against latest ember-data https://github.com/hjdivad/ember-m3/pull/784/checks?check_run_id=808449585#step:5:37